src/styles/cards.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: #{$cards-size-width};
+  padding: 0 16px;
+  margin-bottom: 46px;
+  margin-right: $cards-margin-rights;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__icon {
+    display: block;
+    margin: 32px auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    text-align: left;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__product {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__word {
+    font-weight: 400;
+
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__digits {
+    font-weight: 700;
+
+    font-size: 16px;
+    color: $color-main-accent;
+  }
+
+  &__link {
+    display: flex;
+    margin-bottom: 16px;
+    align-items: center;
+    justify-content: center;
+    padding: 11px 69px 11px 68px;
+
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    color: #fff;
+    text-decoration: none;
+    background: $color-accent-blue;
+    border: 1px solid rgba(255, 255, 255, 0);
+    border-radius: 5px;
+
+    transition: all 2s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```